app/Http/Controllers/API/NodeController.php

```diff
@@ -226,6 +226,33 @@ public function allocations(Request $request)
          }
          return $allocations;
      }
+	 
+    /**
+     * List Node Allocation based on assigned to ID
+     *
+     * Returns a listing of the allocation for the specified server id.
+     *
+     * @Get("/nodes/allocations/{id}")
+     * @Versions({"v1"})
+     * @Response(200)
+     */
+     public function allocations_view(Request $request, $id)
+     {
+        $query = Models\Allocation::where('assigned_to', $id);
+        try {
+            if (!$query->first()) {
+                throw new NotFoundHttpException('No allocation with that ID was found.');
+            }
+    
+            $allocation = $query->first();
```

Comment:
This is just going to return a single allocation for a server, there might be multiple assigned.